docs/message-queues.asciidoc

```diff
@@ -0,0 +1,98 @@
+[[message-quques]]
+== Message Queues
```

Comment:
We use [sentence case capitalization](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case) for headings.

```suggestion
== Message queues
```